Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Create_or_Edit_Security_Policy_Rule.yml

```diff
@@ -0,0 +1,751 @@
+        complex:
+          root: inputs.Application
+          transformers:
+          - args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: any
+            operator: SetIfEmpty
+      category:
+        complex:
+          root: inputs.Category
+      destination:
+        complex:
+          root: inputs.Destination
+      folder:
+        complex:
+          root: inputs.Folder
+          transformers:
+          - args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: Shared
+            operator: SetIfEmpty
+      name:
+        complex:
+          root: inputs.RuleName
+      position:
+        complex:
+          root: inputs.Position
+      service:
+        complex:
+          root: inputs.Service
+          transformers:
+          - args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: any
+            operator: SetIfEmpty
+      source:
+        complex:
+          root: inputs.Source
+          transformers:
+          - args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: any
+            operator: SetIfEmpty
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Create a new security rule.
+      id: d3c57177-a172-4840-86fd-b9d791a247e2
+      iscommand: true
+      name: Create New Security Rule
+      script: '|||prisma-sase-security-rule-create'
+      type: regular
+      version: -1
+    taskid: d3c57177-a172-4840-86fd-b9d791a247e2
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 650,
+          "y": 550
+        }
+      }
+  "5":
+    continueonerrortype: ""
+    id: "5"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 903970f1-cd5d-4ca1-8226-bea09e658349
+      iscommand: false
+      name: Done
+      type: title
+      version: -1
+      description: ''
+    taskid: 903970f1-cd5d-4ca1-8226-bea09e658349
+    timertriggers: []
+    type: title
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1595
+        }
+      }
+  "6":
+    continueonerrortype: ""
+    id: "6"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "7"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      folder:
+        complex:
+          root: inputs.Folder
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Lists all security rules.
```

Comment:
Is this description correct?

Reply:
Yes, that's the default description for this script.